locale/brazil.js

```diff
@@ -0,0 +1,53 @@
+//! moment-holiday.js locale configuration
+//! locale : Brazil
+//! author : Fernando Fabricio dos Santos : https://github.com/ferfabricio
+
+(function() {
+  var moment = (typeof require !== 'undefined' && require !== null) && !require.amd ? require('moment') : this.moment;
+
+  moment.holidays.brazil = {
+    'Ano Novo': {
+      date: '1/1',
+      keywords_n: ['eve']
+    },
+    'Sexta Feira Santa': {
+      date: 'easter-2',
+      keywords_y: ['good', 'friday']
+    },
+    'Domingo de Páscoa': {
+      date: 'easter',
+      keywords_y: ['easter'],
+      keywords: ['sunday']
+    },
+    'Corpus Christi': {
+      date: 'easter+60'
+    },
+    'Tiradentes': {
+      date: '4/21'
+    },
+    'Dia do Trabalhador': {
+      date: '5/1',
+      keywords: ['labour']
+    },
+    'Dia da Independência do Brasil': {
+      date: '8/7',
```

Comment:
right day is: '9/7' or if you are using javascript "getMonth" the correct is: '8/7' and all others dates are wrong (month-1)
https://en.wikipedia.org/wiki/Independence_Day_(Brazil)

Reply:
It was fixed in the commit https://github.com/ferfabricio/moment-holiday/commit/2143172ce5333a890f351c0faeb7ea2f02ef3282

Your revision was in the first commit.

Best regards